### PR TITLE
Self unsharing resource

### PR DESCRIPTION
**Pull request by pkdash**
Could not reproduce the error on self unsharing of a resource. However, it now redirects to my resource page upon unsharing.

### CODE REVIEW

**Comment by pkdash** [comment]
@zhiyuli Can you review this?

**Comment by pkdash** [comment]
@mauriel133 Can you review this? I am using Jquery UI dialog for confirmation of self unsharing of a resource. This modal dialog does not overlay well enough to cover the main navigation bar on the top. I think it's a CSS issue. Let me know how I can fix that.

**Comment by mauriel133** [comment]
I'll have a look.

File: theme/static/css/hydroshare_core.css [hunk 1]
```diff
@@ -4370,6 +4365,12 @@ div#fb-file-operations-controls {
...
-	top: 0px;
-	right:0px;
+	top: 0;
+	right:0;
+}
+
+.ui-dialog {
+	-webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+	box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+	z-index: 1031 !important;
```
Comment (mauriel133) [inline theme/static/css/hydroshare_core.css]: The bootstrap nav bar uses a z-index value of 1030 so I added 1031 for this modal window. If removing the draggable option from the modal window then this line is not needed.

**Comment by mauriel133** [comment]
I added the corresponding z-index to the modal window, however, this drag behavior is buggy and can cause the page to scroll past the footer. I would recommend disabling this by adding  `draggable: false` to the properties.

I also added minor changes for style consistency and mobile devices layout.

**Comment by mauriel133** [comment]
Fixed.

**Comment by pkdash** [comment]
@mauriel133 Thanks. I will merge it then when I see Jenkins happy with the checks.

**Comment by pkdash** [comment]
This is not yet ready to be merged.

**Comment by pkdash** [comment]
@dtarb The PR exposes an error that is hidden in our current deployment. If a non resource owner shares a resource with another user and then tries to revoke that permission this PR shows that an error. The current deployment does not show that an error and gives an impression that the unsharing did work until you refresh the page where you should see the unsharing did not take effect. In my view this needs to be fixed but in another PR. If you agree then I will merge this PR.

**Pull request merged**